Gopkg.toml

```diff
@@ -24,65 +24,67 @@
 #   go-tests = true
 #   unused-packages = true
 
-[[constraint]]
-  name = "cloud.google.com/go"
-  version = "0.34.0"
+# WARNING: IF YOU ARE GETTING WEIRD DEP ERRORS MAKE SURE TO
+# rm -rf test/k8s-integration/src
+# AS IT MAY CONTAIN A PULLED IN KUBERNETES
 
-[[constraint]]
-  name = "github.com/container-storage-interface/spec"
-  version = "1.1.0"
+[prune]
+  go-tests = true
+  unused-packages = true
+  non-go = true
 
-# Transitive dependency of CSI Spec
-[[override]]
-  name = "github.com/golang/protobuf"
-  version = "1.1.0"
+[[constraint]]
+  name = "cloud.google.com/go"
+  version = "0.40.0"
 
 [[constraint]]
   branch = "master"
-  name = "github.com/golang/glog"
-
-#[[constraint]]
-#  name = "github.com/stretchr/testify"
-#  version = "1.2.1"
+  name = "golang.org/x/oauth2"
 
 [[constraint]]
   branch = "master"
-  name = "golang.org/x/net"
+  name = "google.golang.org/api"
 
 [[constraint]]
-  branch = "master"
-  name = "golang.org/x/oauth2"
+  name = "github.com/kubernetes-csi/csi-test"
+  version = "v2.0.1"
 
 [[constraint]]
   branch = "master"
-  name = "google.golang.org/api"
+  name = "k8s.io/test-infra"
+
+###
+### Kubernetes Dependencies
+###
 
 [[constraint]]
-  name = "google.golang.org/grpc"
-  version = "1.10.0"
+  version = "v1.15.0"
+  name = "k8s.io/kubernetes"
 
 [[constraint]]
-  branch = "release-1.11"
+  version = "kubernetes-1.15.0-alpha.3"
```

Comment:
Is there an official 1.15 tag we can use now

Reply:
yeah looks like they were just published